REQUIREMENTS

```diff
@@ -8,3 +8,5 @@ redis>=2.10.3
 requests>=2.7.0
 setuptools>=0.9
 wheel>=0.21
+pika>=0.10.0
```

Comment:
Please put this into a bot-specific REQUIREMENTS, like this one: https://github.com/certtools/intelmq/blob/master/intelmq/bots/outputs/postgresql/REQUIREMENTS.txt